forms/aws/lambda/load_testing/locust_test_mail_list.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import random
+import uuid
+import json
+from json import JSONDecodeError
+from locust import HttpUser, task, between, events
+
+logging.basicConfig(level=logging.INFO)
+
+formIDs = ["81","83"]
+
+formSubmissions = {
+
+  "en":[
+    {
+    "2":{"value":["Immigration program for Afghans who assisted the Government of Canada and their eligible family members."]},
+    "4":{"value":["English"]},
+
+    },
+    {
+    "2":{"value":["Humanitarian program to help Afghan nationals resettle to Canada."]},
+    "4":{"value":["English"]},
+
+    },
+    {
+    "2":{"value":["Immigration program for Afghans who assisted the Government of Canada and their eligible family members."]},
+    "4":{"value":["French"]},
+    },
+    {
+    "2":{"value":["Humanitarian program to help Afghan nationals resettle to Canada."]},
+    "4":{"value":["French"]}
+    }
+  ],
+  "fr":[
+    {
+    "2":{"value":["Programme d’immigration pour les Afghans qui ont aidé le gouvernement du Canada et les membres admissibles de leurs familles."]},
+    "4":{"value":["Anglais"]},
+    },
+    {
+    "2":{"value":["Programme humanitaire pour aider les afghans à s’installer au Canada."]},
+    "4":{"value":["Anglais"]},
+    },
+    {
+    "2":{"value":["Programme d’immigration pour les Afghans qui ont aidé le gouvernement du Canada et les membres admissibles de leurs familles."]},
+    "4":{"value":["Français"]},
+    },
+    {
+    "2":{"value":["Programme humanitaire pour aider les afghans à s’installer au Canada."]},
+    "4":{"value":["Français"]}
+    }
+  ]
+}
+
+def generatePhoneorEmail(formID):
+  if formID == "81":
+    return uuid.uuid4().hex + "@cds-snc.ca"
```

Comment:
change to success simulator email success@simulator.amazonses.com

Reply:
Also were not going to test phone as there is nothing that allows us to simulate